### DISCUSSION TITLE
質問回答についたコメントの通知のリンク先をそのコメントにして欲しい

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#501] [要望・アイデア・不具合]
質問回答した内容にコメントが付くと通知アイコンにリンクが表示されますが、リンク先が自分が質問回答した内容の先頭になっていて、コメントされた内容を表示するのにスクロールしたり畳まれたコメントを広げてさらにスクロールしたり手間がかかっています。
通知をクリックしたら質問回答のコメントに直接飛ぶようにしていただけないでしょうか。

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご意見ありがとうございます！
確かに直接コメントに飛べた方がいいと思いますので、今後どこかのタイミングで改善したいと思います🙇

**Comment by tanaka-asami415** [comment]
@shiracamus 
お待たせしました🙏
通知リンクより「質問回答についたコメント」へ直接飛ぶことができるようになりました！

別ページや通知メールから遷移していただく場合には、コメントは開かれた状態で確認していただけるのですが
コメントがついた質問ページを開いている状態で通知欄のリンクをクリックされた場合は
現在のQiitaの仕様上、コメントは折りたたまれたままになってしまうためリロードが必要になります🙏

こちらも将来的には対応したいと思いますが、一旦現状で対応できる形でリリースいたしました！🙇
少しでもお手間が軽減されると幸いです✨
